init.lua

```diff
@@ -197,7 +197,7 @@ local function smooth(pos1, pos2, deadzone)
 	local offset = vector.subtract(pos1, area.MinEdge)
 	local c_air = minetest.get_content_id("air")
 	local c_dirt = minetest.get_content_id("default:dirt")
-
+	
```

Comment:
whitespace

Reply:
Erm you want it removed? Uhhh sure? Not my preference (I think my editor did it automagically), but I can change it here.